src/tribler/core/sentry_reporter/sentry_reporter.py

```diff
@@ -18,9 +18,7 @@
     delete_item,
     get_first_item,
     get_last_item, get_value,
-    parse_last_core_output, parse_os_environ,
-    parse_stacktrace,
-)
+    parse_last_core_output, )
```

Comment:
Formatting: unnecessary comma
```suggestion
    parse_last_core_output
)
```